src/rust/madara/sources/manganinja/res/source.json

```
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "pt-br.manganinja",
+		"lang": "pt-br",
+		"name": "Manga Ninja",
+		"version": 1,
+		"url": "https://manganinja.com",
+		"nsfw": 2
```

this doesn't look like it has nsfw?
```suggestion
		"nsfw": 0
```

---

I verified and you're right, there's no nsfw.